pages/pipelines/hosted_agents/linux.md

```diff
@@ -28,13 +28,90 @@ To configure your Linux instance you can use the [Docker Compose](https://github
 
 ## Cache volumes
 
-> 🚧 Under development
-> This feature is currently not available.
+Cache Volumes are external volumes attached to hosted agent instances. Cache volumes are attached on best-effort basis depending on their locality, expiration and current usage (so they should not be relied upon as durable data storage).
```

Comment:
```suggestion
Cache Volumes are external volumes attached to hosted agent instances. Cache volumes are attached on a best-effort basis depending on their locality, expiration and current usage (so they should not be relied upon as durable data storage).
```

Reply:
I'm assuming that we want to use Title Case for Cache Volumes as its a feature? https://github.com/buildkite/docs/blob/main/styleguides/writing-style.md#product-names-and-product-features